docker/rstudio/Dockerfile

```diff
@@ -21,6 +21,11 @@ RUN R -e 'install.packages(c( \
     repos="http://cran.mtu.edu")' \
  && R -e 'devtools::install_github("DataBiosphere/Ronaldo")'
 
+# install python dev libs
+RUN apt-get update
+RUN apt-get install -y libpython-dev
+RUN apt-get install -y libpython3-dev
```

Comment:
Does this include `pip`? Another way to install it is:
https://pip.pypa.io/en/stable/installing/

Reply:
no, but it makes pip easily installable at the very least. i'll add it to the image